onnx2torch/node_converters/activations.py

```diff
@@ -48,6 +49,28 @@ def forward(self, input_tensor: torch.Tensor) -> torch.Tensor:  # pylint: disabl
         return torch.reshape(result, shape)
 
 
+class OnnxPReLU(nn.Module, OnnxToTorchModuleWithCustomExport):  # pylint: disable=missing-docstring
+    def forward(  # pylint: disable=missing-function-docstring
+        self,
+        input_tensor: torch.Tensor,
+        slope: torch.Tensor,
+    ) -> torch.Tensor:
+        output = input_tensor.clone()
+        output = output * slope
+        mask = input_tensor > 0
```

Comment:
I think here should be >=.

PRelu takes input data (Tensor) and slope tensor as input, and produces one output data (Tensor) where the function f(x) = slope * x for x < 0, **f(x) = x for x >= 0**., is applied to the data tensor elementwise.